cli/scripts/e2e/e2e.ts

```diff
@@ -85,15 +85,46 @@ function runSmokeTests<T>(
   );
 
   suite(
-    `${npmClient} handles filesystem changes ${
+    `${npmClient} handles filesystem changes${
       options.cwd ? " from " + options.cwd : ""
-    } `,
+    }`,
     async () => {
       repo.newBranch("my-feature-branch");
       repo.commitFiles({
         [path.join("packages", "a", "test.js")]: `console.log('testingz a');`,
       });
 
+      const sinceCommandOutputNoCache = getCommandOutputAsArray(
+        repo.turbo(
+          "run",
+          ["test", "--since=main", "--stream", "--no-cache"],
+          options
+        )
+      );
+      assert.fixture(
+        `• Packages changed since main: a`,
+        sinceCommandOutputNoCache[0],
+        "Calculates changed packages (--since)"
+      );
+      assert.fixture(
+        `• Packages in scope: a`,
+        sinceCommandOutputNoCache[1],
+        "Packages in scope"
+      );
+      assert.fixture(
+        `• Running test in 1 packages`,
+        sinceCommandOutputNoCache[2],
+        "Runs only in changed packages"
+      );
+      assert.fixture(
+        sinceCommandOutputNoCache[3],
+        `a:test: cache miss, executing ${getHashFromOutput(
```

Comment:
This isn’t fully testing —no-cache. Need to assert that the outputted log file was not copied into the cache folder

Reply:
Isn't that assumed if https://github.com/vercel/turborepo/blob/d81311665f337788ba9459f5173c671f12214cb2/cli/scripts/e2e/e2e.ts#L128 is a cache miss?

Reply:
ahh true that. :smart: